### DISCUSSION TITLE
What values are expected for e.g. AccessId and AccessSecret when using ghcr?

### DISCUSSION THREAD

**Discussion by devantler** [goharbor/harbor#20611] [Q&A] [answered]
I am trying to configure a pull-through ghcr registry for my private packages on GitHub. I am unsure what values are expected in AccessId and AccessSecret.

Has anyone successfully configured such a setup?

**Comment by MinerYang** [comment, marked as answer]
It should be your `username` and `password` for your GHCR

**Reply by devantler** [reply]
Thanks! This was the case, I had some other issues, that made it look like my auth was not working between Harbor and GHCR, when in reality the issue was between the cluster and Harbor. It was partly overconfidence 😋
